Mathlib/Lean/SourceInfo_fromRef.lean

```diff
@@ -0,0 +1,19 @@
+/-
+Copyright (c) 2022 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+
+import Lean
+
```

Comment:
If you merge master, the linter will probably complain here about a missing module doc.

Reply:
Added module docs, saying that this stuff is supposed to move to core.